lib/utils/logger.js

```diff
@@ -83,26 +100,59 @@ const log = method => (...args) => {
 		const frame = trace && trace[2];
 		const str = JSON.stringify(frame) || '';
 
-		prefix += ' ' + str + '\n';
+		prefix += ' ' + str + EOL;
 	}
 
 	// log to console
-	console[method.consoleMethod](colors[method.color](prefix, ...args));
+	if (consoleType === 'lib') {
+		console[method.consoleMethod](colors[method.color](prefix, ...args));
+	} else if (consoleType === 'launcher') {
+		console[method.consoleMethod](method.launcherColor
+			? method.launcherColor(devicePrefix(args[1]), args[0])
+			: stColors.suit(devicePrefix(args[1])) + args[0]
+		);
+	} else {
+		console[method.consoleMethod](colors[method.color](...args));
+	}
 };
 
 /**
- * Create logger object
+ * Create logger objects
  */
-const logger = {
+const libLogger = {
 	levels: {...logLevels},
+	colors: stColors,
 };
 
+const launcherLogger = {
+	colors: stColors,
+};
+
+/**
+ * Prefix log with device id
+ * @param {string} devId
+ */
+function devicePrefix(devId) {
+	return devId ? 'Device ' + devId.slice(0, 8) + '> ' : '               > ';
```

Comment:
How much spaces in value from false branch? You can use something like `'> '.padStart(10, ' ')`